StencilSwiftKit.xcodeproj/project.pbxproj

```diff
@@ -1,463 +1,1570 @@
-				DEBUG_INFORMATION_FORMAT = "dwarf-with-dsym";
-				ENABLE_NS_ASSERTIONS = NO;
-				ENABLE_STRICT_OBJC_MSGSEND = YES;
-				GCC_C_LANGUAGE_STANDARD = gnu99;
-				GCC_NO_COMMON_BLOCKS = YES;
-				GCC_WARN_64_TO_32_BIT_CONVERSION = YES;
-				GCC_WARN_ABOUT_RETURN_TYPE = YES_ERROR;
-				GCC_WARN_UNDECLARED_SELECTOR = YES;
-				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
-				GCC_WARN_UNUSED_FUNCTION = YES;
-				GCC_WARN_UNUSED_VARIABLE = YES;
-				MACOSX_DEPLOYMENT_TARGET = 10.10;
-				MTL_ENABLE_DEBUG_INFO = NO;
-				SDKROOT = macosx;
-				SWIFT_OPTIMIZATION_LEVEL = "-Owholemodule";
-				SWIFT_VERSION = 2.3;
-			};
-			name = Release;
-		};
-		09A87B561BCCA2C600D9B9F5 /* Debug */ = {
-			isa = XCBuildConfiguration;
-			baseConfigurationReference = 47888DD528DEC4C84FD8F15B /* Pods-Tests.debug.xcconfig */;
-			buildSettings = {
-				COMBINE_HIDPI_IMAGES = YES;
-				INFOPLIST_FILE = Tests/StencilSwiftKitTests/Resources/Info.plist;
-				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/../Frameworks @loader_path/../Frameworks";
-				MACOSX_DEPLOYMENT_TARGET = 10.11;
-				PRODUCT_BUNDLE_IDENTIFIER = com.alisoftware.SwiftGenKitTests;
-				PRODUCT_NAME = "$(TARGET_NAME)";
-				SWIFT_VERSION = 4.2;
-			};
-			name = Debug;
-		};
-		09A87B571BCCA2C600D9B9F5 /* Release */ = {
-			isa = XCBuildConfiguration;
-			baseConfigurationReference = 4B3D39DBCD15D8F6BB891D92 /* Pods-Tests.release.xcconfig */;
-			buildSettings = {
-				COMBINE_HIDPI_IMAGES = YES;
-				INFOPLIST_FILE = Tests/StencilSwiftKitTests/Resources/Info.plist;
-				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/../Frameworks @loader_path/../Frameworks";
-				MACOSX_DEPLOYMENT_TARGET = 10.11;
-				PRODUCT_BUNDLE_IDENTIFIER = com.alisoftware.SwiftGenKitTests;
-				PRODUCT_NAME = "$(TARGET_NAME)";
-				SWIFT_VERSION = 4.2;
-			};
-			name = Release;
-		};
-/* End XCBuildConfiguration section */
-
-/* Begin XCConfigurationList section */
-		09030B661B6D254200275BF5 /* Build configuration list for PBXProject "StencilSwiftKit" */ = {
-			isa = XCConfigurationList;
-			buildConfigurations = (
-				09030B701B6D254200275BF5 /* Debug */,
-				09030B711B6D254200275BF5 /* Release */,
-			);
-			defaultConfigurationIsVisible = 0;
-			defaultConfigurationName = Release;
-		};
-		09A87B551BCCA2C600D9B9F5 /* Build configuration list for PBXNativeTarget "Tests" */ = {
-			isa = XCConfigurationList;
-			buildConfigurations = (
-				09A87B561BCCA2C600D9B9F5 /* Debug */,
-				09A87B571BCCA2C600D9B9F5 /* Release */,
-			);
-			defaultConfigurationIsVisible = 0;
-			defaultConfigurationName = Release;
-		};
-/* End XCConfigurationList section */
-	};
-	rootObject = 09030B631B6D254200275BF5 /* Project object */;
+   archiveVersion = "1";
+   objectVersion = "46";
+   objects = {
+      "OBJ_1" = {
+         isa = "PBXProject";
+         attributes = {
+            LastSwiftMigration = "9999";
+            LastUpgradeCheck = "9999";
+         };
+         buildConfigurationList = "OBJ_2";
```

Comment:
Seems like a strange diff for a pbxproj, with the list of `objects` being of the form `OBJ_nn` instead of being UUIDs like usual, and with `LastSwiftMigration` set to `9999`, etc…

How was this xcodeproj generated/updated? (Is it twist-generated? If so, would be better to do such a change in a separate PR to keep the diff focused on just the update of Stencil to 0.14)

Reply:
Yeah, I thought the `.xcodeproj` was being created with swift package manager - which turns out not to be the case 😄 I'll revert the change. Thanks for the pointers, I'll ping you once I feel this PR is ready to be reviewed/I get stuck again 🙂